style.css

```diff
@@ -0,0 +1,376 @@
+    url("assets/chomsky-8moa2-webfont.woff2") format("woff");
+  font-weight: normal;
+  font-style: normal;
+}
+
+#wrapper {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 40% 1fr 1fr 1fr;
+  grid-auto-rows: minmax(100px, auto);
+  grid-gap: 1px;
+  max-width: 100vw;
+  margin: 0 auto;
+  grid-template-areas:
+    "nav nav nav nav nav nav nav"
+    "l-aside . main main main . ."
+    "l-aside . main main main r-aside r-aside"
+    ". . main main main . ."
+    ". . footer-images footer-images footer-images . ."
+    ". . footer footer footer . .";
+}
+
+.footer-images {
+  grid-area: footer-images;
+}
+
+main,
+aside,
+section,
+footer {
+  padding: 10px;
+  font-family: "Times New Roman", Times, serif;
+}
+
+main {
+  grid-area: main;
+}
+
+.left-aside {
+  grid-area: l-aside;
+}
+
+.social {
+  display: flex;
+  justify-content: space-between;
+  padding-top: 10px;
+}
+
+.comment > * {
+  width: 250px;
+  height: auto;
+  background-color: cadetblue;
+  color: white;
+  border: transparent;
+  border-radius: 3px;
+  text-align: center;
+}
+
+.social-icons > * {
+  margin-left: 5px;
+}
+
+nav {
+  background-color: white;
+  border-bottom: solid gray;
+  border-width: 1px;
+  padding: 10px;
+  width: 100%;
+  height: auto;
+  grid-area: nav;
+  display: flex;
+  justify-content: space-evenly;
+  position: fixed;
+}
+
+.nav-button {
+  padding: 3px;
+  max-width: 130px;
+  font-size: 70%;
+  background-color: cadetblue;
+  border-radius: 10px;
+  color: cornsilk;
+  font-family: "Times New Roman", Times, serif;
+}
+
+.onclick {
+  background-color: white;
+  border: transparent;
+  margin-left: 5px;
+}
+
+.nav-search {
+  display: none;
+}
+
+.onclick:focus + .nav-search {
+  display: inline;
+}
+
+#nytimes-logo {
+  font-family: "chomskyregular", Arial, sans-serif;
+  font-size: 100%;
+  font-weight: 400;
+}
+
+section {
+  grid-area: section;
+}
+
+footer {
+  grid-area: footer;
+}
+
+.right-aside {
+  grid-area: r-aside;
+  font-size: 100%;
+  max-width: 100%;
+  float: left;
+}
+
+.title-header {
+  font-style: italic;
+  max-width: 50%;
+  margin: 0 auto;
+}
+
+.main-img,
+.middle-img {
+  max-width: 100%;
+  max-height: 30%;
+  margin-top: 10px;
+}
+
+article {
+  max-width: 600px;
+  margin: 20px auto;
+}
+
+article p {
+  margin: 20px 0;
+}
+
+.diagrams {
+  max-width: 100%;
+  margin: 0;
+}
+
+.diagrams-container {
+  display: grid;
+  grid-template-columns: repeat(4, 4fr);
+  grid-template-rows: repeat(4, 1fr);
+  grid-auto-flow: column;
+  gap: 2px;
+  width: 100%;
+  margin-top: 10px;
+}
+
+.bottom-section img {
+  width: 100%;
+}
+
+.left-b-section img {
+  object-fit: contain;
+}
+
+.r-flex-box > img {
+  max-width: 130px;
+  margin-right: 10px;
+}
+
+.diagrams-container p img {
+  width: 90%;
+}
+
+.bottom-section {
+  width: 100%;
+  margin: auto;
+  border-bottom: 1px solid black;
+  font-size: 100%;
+}
+
+.left-b-section {
+  width: 70%;
+  display: flex;
+  flex-wrap: wrap;
```

Comment:
## Using the Same Rule Repetitively[1]
- [ ] Create a class once and the use in each HTML tag where you need it.
- [ ] Delete all your `display:flex;` properties in entire CSS file
- [ ] Add your HTML tags

**CSS file**

```
.flex{
  display:flex;
}
```
**Html file**

```
<div class="flex">
```

<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_